src/providers/siliconflow/imageGenerate.ts

```diff
@@ -0,0 +1,47 @@
+import { SILICONFLOW } from '../../globals';
+import { ErrorResponse, ImageGenerateResponse, ProviderConfig } from '../types';
+import { SiliconFlowErrorResponseTransform } from './chatComplete';
+
+export const SiliconFlowImageGenerateConfig: ProviderConfig = {
+  prompt: {
+    param: 'prompt',
+    required: true,
+  },
+  image_size: {
```

Comment:
We take the input in OpenAI format. So this should take size as input and map it to image_size